src/config/config.ts

```diff
@@ -300,6 +304,21 @@ export function initializeServerConfiguration(rawConfiguration: any) {
             console.log('Error parsing localStorage.frontendConfig');
         }
     }
+    if (localStorage.oncoprintDefaultTracksConfigOverride) {
+        try {
+            rawConfiguration.oncoprint_clinical_tracks_show_by_default = JSON.parse(
+                localStorage.oncoprintDefaultTracksConfigOverride
+            );
+            console.log(
+                'Using localStorage.oncoprintDefaultTracksConfigOverride (overriding rawConfiguration.oncoprintDefaultTracksConfigOverride): ' +
+                    localStorage.oncoprintDefaultTracksConfigOverride
+            );
+        } catch (err) {
+            console.log(
+                'Error parsing localStorage.oncoprintDefaultTracksConfigOverride'
+            );
+        }
+    }
```

Comment:
Different from the `frontendConfig` property, the oncoprint config does not have to be put in the localStorage. In that respect you should use the `query_sets_of_genes` property as an example. This frontend property is named `querypage.setsofgenes.location` in portal properties.

Reply:
fixed